server/scsynth/SC_ComPort.cpp

```diff
@@ -356,13 +356,13 @@ class SC_TcpConnection:
 		data = (char*)malloc(OSCMsgLength);
 
 		async_read(socket, ba::buffer(data, OSCMsgLength),
-				   boost::bind(&SC_TcpConnection::handleMsgReceived, shared_from_this(),
-							   ba::placeholders::error,
-							   ba::placeholders::bytes_transferred));
+		           boost::bind(&SC_TcpConnection::handleMsgReceived, shared_from_this(),
+		           ba::placeholders::error,
+		           ba::placeholders::bytes_transferred));
```

Comment:
Same as lines 331-334 above. The outer close paren should also be moved for consistency (while I've got my pedantry cap on)